cliff.toml

```diff
@@ -30,17 +34,40 @@ body = """
     ### 📃 Details
 {% for group, commits in commits | group_by(attribute="group") %}
     #### {{ group | upper_first }}
-    {% for commit in commits %}
-        * {{ commit.message | upper_first }}\
+    {%- for commit in commits %}
+        - {{ commit.message | upper_first | trim }}\
+            {% if commit.github.username %} by @{{ commit.github.username }}{%- endif -%}
+            {% if commit.github.pr_number %} in #{{ commit.github.pr_number }}{%- endif -%}
```

Comment:
I try to add the full links to the PRs instead of just #<number> to ensure that the changelog also works on other websites besides github.com

Reply:
Done in 50b4c03b7da42227813ff4a6c8320fcd69d448f3